src/ports/nfts/utils.ts

```diff
@@ -199,6 +199,19 @@ export function getFetchQuery(
 
   addEmoteCategoryAndRaritiesFilters(filters, where)
 
+  if (filters.adjacentToRoad) {
```

Comment:
Should we add `filters.isLand` here as well?
```suggestion
  if (filters.isLand && filters.adjacentToRoad) {
```